Gemfile

```diff
@@ -1,12 +1,60 @@
 source("https://rubygems.org")
 
-gem "xcode-install", ">= 2.6.7" # needed for running xcode-install related tests
+# Needed for the Fastlane::RUBOCOP_REQUIREMENT below
+require_relative "fastlane/lib/fastlane/version.rb"
 
+# Please don't add line breaks between the gems and their comments, as RuboCop won't be able to sort these alphabetically.
+
+# Allows fine-grained control of environment variables.
+gem "climate_control", "~> 0.2.0"
+# A tool for integrating Coveralls.io with Ruby apps.
+gem "coveralls", "~> 0.8.13"
+# Automates code review chores.
 gem "danger", "~> 8.0"
+# Plugin for Danger that reports JUnit test results.
 gem "danger-junit", "~> 1.0"
+# A fake filesystem.
+# Version 1.9+ requires Ruby >=2.7, while fastlane uses a `required_ruby_version` of `>= 2.6`.
+gem "fakefs", "1.8"
+# Fast XML parser and object marshaller.
 gem "ox", "2.13.2"
+# Provides an interactive debugging environment for Ruby.
+gem "pry"
+# A plugin for pry that adds step-by-step debugging and stack navigation.
+gem "pry-byebug"
+# A pry rescue environment to automatically open pry when a test fails.
+gem "pry-rescue"
+# A plugin for pry that enables exploring the call stack.
+gem "pry-stack_explorer"
+# A simple task automation tool.
+gem "rake"
+# A readline implementation in Ruby
+# See: https://github.com/deivid-rodriguez/byebug/issues/289#issuecomment-251383465
+gem "rb-readline"
+# A simple and flexible REST client.
+gem "rest-client", ">= 1.8.0"
+# Behavior-driven testing tool for Ruby.
+gem "rspec", "~> 3.10"
+# Formatter for RSpec to generate JUnit compatible reports.
+gem "rspec_junit_formatter", "~> 0.4.1"
+# A Ruby static code analyzer and formatter.
+gem "rubocop", Fastlane::RUBOCOP_REQUIREMENT
+# A collection of RuboCop cops for performance optimizations.
+gem "rubocop-performance"
+# A RuboCop extension focused on enforcing tools.
+gem "rubocop-require_tools"
+# Used to mock servers.
+gem "sinatra", "~> 2.0.8"
+# A library for stubbing and setting expectations on HTTP requests.
+gem "webmock", "~> 3.18"
+# Needed for running xcode-install related tests.
+gem "xcode-install", ">= 2.6.7"
+# Used for xcov's parameters generation: https://github.com/fastlane/fastlane/pull/12416
+gem "xcov", "~> 1.4.1"
+# A documentation generation tool for Ruby.
+gem "yard", "~> 0.9.11"
 
 gemspec(path: ".")
 
-plugins_path = File.join(File.expand_path("..", __FILE__), 'fastlane', 'Pluginfile')
+plugins_path = File.join(File.expand_path("..", __FILE__), "fastlane", "Pluginfile")
```

Comment:
why changing the quote style?

Reply:
@lacostej Just converged a previously inconsistent style to a single quote style within the file, for consistency sake 😅 

And between single quote and double quote, since our rubocop config and style guide allows for both, I picked my preferred choice, but happy to revert them all to be single quoted, as long as they're all consistent 😅 

My reasoning for double quotes is mostly this:

> You also have to change the quote style when you add or remove interpolation from an existing string. If you always use double quotes then you never have to waste any time thinking about it.

Taken from: https://anti-pattern.com/always-use-double-quoted-strings-in-ruby

Reply:
Also, not only my personal preference, but double quotes was the predominant style used in this file already 😄

Reply:
I was curious because the other PR came with single quote in the Gemfile

https://github.com/fastlane/fastlane/pull/21726/files#diff-1980c6aefca6af4ba5d92c31aa38a037210a2bbd76f8215657e6cebdc2c1def5

I personally don't mind using double quotes everywhere.